internal/sdkprovider/service/clickhouse/grant.go

```diff
@@ -145,12 +145,14 @@ func createPrivilegeGrantStatement(grant PrivilegeGrant) string {
 
 	if len(grant.Table) > 0 {
 		b.WriteString(fmt.Sprintf(".%s", escape(grant.Table)))
+	} else {
+		b.WriteString(".*")
```

Comment:
perhaps we want to have something similar at line 176 too?

Reply:
We do :+1: